HTML/Exercise5/script.js

```diff
@@ -0,0 +1,134 @@
+const search = document.querySelector(".query");
+const container = document.querySelector(".container");
+const select = document.getElementById("update");
+const rangep = document.getElementById("range");
+const update = document.getElementById("update");
+const load = document.getElementById("load");
+
+update.addEventListener("change", (e) => sortData(e.target.value));
+
+let data = [];
+
+let pageCount = 1;
+
+let years = []; // all fetched years from response
+
+let btn = document.getElementById("unique");
+let loader = false;
+
+btn.addEventListener("click", fetchData());
+
+async function fetchData() {
+  container.innerHTML = "";
+  load.innerText = "Loading please wait...";
+  let fragment = document.createDocumentFragment();
+  let fragment2 = document.createDocumentFragment();
+  years = [];
+
+  if (search.value === "") return;
+  for (let i = 1; i < 100; i++) {
+    data = await (
+      await fetch(
+        `https://www.omdbapi.com/?apikey=d5a4ba2c&s=${search.value}&page=${i}`
```

Comment:
ideally, you'd use a env variable but here you could make a variable to store the baseURL/apikey since it's used multiple times

Reply:
Done